src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,20 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
+            logger.info("User with username '{}' has successfully logged in.", user.getLogin());
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Failed to authenticate user: {}", e.getMessage(), e);
```

Comment:
Let's make our log messages informative. Include the username in the log message to make it more informative. For example: logger.error("Failed to authenticate user with login='{}': {}", login, e.getMessage(), e);